packages/sdk/src/dataProtector/sharing/setProtectedDataAsRentable.ts

```diff
@@ -0,0 +1,65 @@
+import { WorkflowError } from '../../utils/errors.js';
+import {
+  isCollectionOwner,
+  isProtectedDataInCollection,
+} from '../../utils/sharing.js';
+import { throwIfMissing } from '../../utils/validators.js';
+import {
+  IExecConsumer,
+  SetProtectedDataAsRentableParams,
+  SetProtectedDataAsRentableResponse,
+  SubgraphConsumer,
+} from '../types.js';
+import { getSharingContract } from './smartContract/getSharingContract.js';
+
+export const setProtectedDataToRenting = async ({
```

Comment:
```suggestion
export const setProtectedDataAsRentable = async ({
```

to match the file name?